### ISSUE TITLE
Rotating pitch/yaw of a page and its icons causes blur and performance loss

### ISSUE THREAD

**Issue by rweichler** [rweichler/cylinder#17]
no idea why.

**Comment by rweichler** [comment]
@noahd  did you end up fixing this in your branch?

**Comment by noahd** [comment]
@rweichler Yeah, in principle, *that* bug I fixed early on, only other bugs surfaced (that I don't think exist in the master branch), like some icons randomly vanishing after unlocking. They're not that annoying and only seem to happen seldomly, but I kinda wanted to squash them before making a PR and eventually I Iost interest. 😐

In the summer, I actually worked on another solution that doesn't require weirdly messing with the view hierarchy: 'buffering' whatever page layer transforms the scripts want (but not actually applying them), and then applying the page transform on each icon, combined with its own respective transform. That didn't really work, probably because I got the math wrong (and then I stopped working on it because college), but it might be worth another try.

**Comment by rweichler** [comment]
The delegation of transformation from the page to the individual icons is an interesting.... but CATransformLayer would be SOOO much easier..
